apps/web/app/(use-page-wrapper)/apps/routing-forms/[...pages]/page.tsx

```diff
@@ -48,15 +48,16 @@ const ServerPage = async ({ params, searchParams }: ServerPageProps) => {
     </FormProvider>
   );
 
-  if (mainPage === "routing-link") {
-    return <FinalComponent />;
+  // TODO: we should really be using layout for this.
+  if (mainPage === "forms") {
```

Comment:
Let's add an inline comment here that we don't want Shell for 
Routing Form Edit and Routing Form booking page